js/main.js

```diff
@@ -33,3 +39,50 @@ submitForm.addEventListener('submit', handleSubmit);
 // Prepend the new object to the entries in the data model.
 // Reset the image preview's `src' attribute.
 // Reset the form inputs
+
+function createEntry(e) {
+  // create a li, create two divs. place both divs within the li
+  var entryItem = document.createElement('li');
+  entryItem.className = 'row entry-li';
+  var entryImgDiv = document.createElement('div');
+  var entryImg = document.createElement('img');
+  entryImg.className = 'image-input';
+  entryImg.src = e.photoUrl;
+  entryImg.alt = e.title + ' Image';
+  entryImgDiv.className = 'column-half';
+  var entryTextDiv = document.createElement('div');
+  var entryTextTitle = document.createElement('h4');
+  var entryTextNotes = document.createElement('p');
+  entryTextNotes.textContent = e.notes;
+  entryTextTitle.textContent = e.title;
+  entryTextDiv.className = 'column-half entry-text';
+  entryItem.appendChild(entryImgDiv);
+  entryItem.appendChild(entryTextDiv);
+  entryImgDiv.appendChild(entryImg);
+  entryTextDiv.appendChild(entryTextTitle);
+  entryTextDiv.appendChild(entryTextNotes);
+  return entryItem;
+}
+var userEntries = document.querySelector('.entries-ul');
+
+function loadUserEntries(e) {
+  for (var i = 0; i < data.entries.length; i++) {
+    userEntries.appendChild(createEntry(data.entries[i]));
+  }
+}
+document.addEventListener('DOMContentLoaded', loadUserEntries);
+
+var newEntryButton = document.querySelector('[name="new-entry"]');
+var newEntryContainer = document.querySelector('.new-entry-container');
+
+function displayNewEntry(e) {
+  if (newEntryContainer.style.display === 'none') {
+    newEntryContainer.style.display = 'block';
+    newEntryButton.textContent = 'Close';
```

Comment:
as noted in the feedback, make sure to avoid inline styling. The specificity of inline styling is very high and all CSS should be done in the CSS files.